components/CompetitionInfo.tsx

```diff
@@ -0,0 +1,117 @@
+import { Card, CardContent, Typography } from "@mui/material";
+import React from "react";
+
+const CompetitionInfo = () => {
+  return (
+    <Card sx={{ mt: 7, borderTop: "3px solid", borderColor: "primary.main" }}>
```

Comment:
```suggestion
    <Card
      sx={{
        mt: 7,
        borderTop: "3px solid",
        borderColor: "primary.main",
        maxWidth: "1200px",
      }}
    >
```

Add a max width, on large screens it gets too wide
<img width="1792" alt="Screenshot 2022-03-07 at 08 11 24" src="https://user-images.githubusercontent.com/8343002/156989210-75bfc3c9-80b4-453d-8a3a-c3b0a5c69de3.png">

Reply:
Totally agree that this is needed!

Reply:
Fixed in #95 :)